governance/votes/Executive vote - April 18, 2024.md

```diff
@@ -0,0 +1,128 @@
+
+The Governance Facilitators, SideStream, Dewiz, and Phoenix Labs have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following actions will occur within the Maker Protocol:
+
+- A total of **219.36 MKR** will be distributed to eight Aligned Delegates as compensation for March 2024.
+- A total of **175.14 MKR** will be distributed to nine AVC members as participation rewards for Q1 2024.
+- A total of **238,339 DAI** will be paid to Aave as part of the SparkLend revenue share.
+- A new address in the RWA015-A output conduit will be whitelisted.
+- **84,211.27 USDP** will be pushed out of the MCD PSM PAX Input Conduit Jar.
+- A proxy spell for Spark Protocol on Ethereum and Gnosis will be triggered.
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **30 hours**.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### Aligned Delegate Compensation for March 2024
+
+- **Authorization:** [Atlas Section 2.6.3](https://mips.makerdao.com/mips/details/MIP101#2-6-3-aligned-delegate-income-and-participation-requirements)
+- **Proposal:** [Forum post](https://forum.makerdao.com/t/march-2024-aligned-delegate-compensation/24088)
+
+If this executive proposal passes, the following transfers totalling **214.55 MKR** for Aligned Delegate Compensation will be made to qualifying Aligned Delegates.
+
+| Delegate | Address | Amount (MKR) |
+|--|--|--|
+| BONAPUBLICA | 0x167c1a762B08D7e78dbF8f24e5C3f1Ab415021D3 | 41.67 |
+| Cloaky | 0x869b6d5d8FA7f4FFdaCA4D23FFE0735c5eD1F818 | 41.67 |
+| TRUE NAME | 0x612F7924c367575a0Edf21333D96b15F1B345A5d | 41.67 |
+| BLUE | 0xb6C09680D822F162449cdFB8248a7D3FC26Ec9Bf | 39.75 |
+| vigilant | 0x2474937cB55500601BCCE9f4cb0A0A72Dc226F61 | 13.89 |
+| Pipkin | 0x0E661eFE390aE39f90a58b04CF891044e56DEDB7 | 13.89 |
+| JAG | 0x58D1ec57E4294E4fe650D1CB12b96AE34349556f | 9.08 |
+| UPMaker | 0xbB819DF169670DC71A16F58F55956FE642cc6BcD | 12.93 |
```

Comment:
:green_circle: **general comment, no action required**

In general we feel that these distributions should be ordered by amount to be distributed (largest-to-smallest) and then by Delegate Name (A-Z).

However, this order matches the sheet which is what the code will be based on so we will not suggest a change at this time in order that it matches the code for ease of review.